modules/data.atmosphere/R/met2CF.Drought2018.R

```diff
@@ -0,0 +1,52 @@
+#' Convert variables ICOS Drought 2018 variables to CF format. 
+#' 
+#' Variables present in the output netCDF file:
+#' air_temperature, air_temperature, relative_humidity,
+#' specific_humidity, water_vapor_saturation_deficit,
+#' surface_downwelling_longwave_flux_in_air,
+#' surface_downwelling_shortwave_flux_in_air,
+#' surface_downwelling_photosynthetic_photon_flux_in_air, precipitation_flux,
+#' eastward_wind, northward_wind
+#'
+#' @param in.path path to the input Drought 2018 CSV file
+#' @param in.prefix name of the input file
+#' @param outfolder path to the directory where the output file is stored. If specified directory does not exists, it is created.
+#' @param start_date start date of the input file
+#' @param end_date end date of the input file
+#' @param format format is data frame or list with elements as described below
+#'   REQUIRED:
+#'   format$header = number of lines of header
+#'   format$vars is a data.frame with lists of information for each variable to read, at least airT is required
+#'     format$vars$input_name = Name in CSV file
+#'     format$vars$input_units = Units in CSV file
+#'     format$vars$bety_name = Name in BETY
+#'   OPTIONAL:
+#'   format$lat = latitude of site
+#'   format$lon = longitude of site
+#'   format$na.strings = list of missing values to convert to NA, such as -9999
+#'   format$skip = lines to skip excluding header
+#'   format$vars$column_number = Column number in CSV file (optional, will use header name first)
+#' Columns with NA for bety variable name are dropped.
+#' @param overwrite overwrite should existing files be overwritten. Default False.
+#' @return information about the output file
+#' @export
+#'
+
+met2CF.Drought2018 <-
```

Comment:
Just to followup on the discussion above regarding generic `download.ICOS`, if we go that way, this function would be renamed to `met2CF.ICOS` and here you may deal with different formats and/or missing variables potentially (similar to `met2CF.AmerifluxLBL`)